contrib/grinder/scripts/ingestenum.py

```diff
@@ -0,0 +1,86 @@
+import random
+import time
+try:
+  from com.xhaus.jyson import JysonCodec as json
+except ImportError:
+  import json
+from utils import *
+from net.grinder.script import Test
+from net.grinder.plugin.http import HTTPRequest
+
+class EnumIngestThread(AbstractThread):
+  # The list of metric numbers for all threads in this worker
+  metrics = []
+
+  # Grinder test reporting infrastructure
+  test1 = Test(3, "Enum Ingest test")
```

Comment:
@VinnyQ I think this test number is not unique, which is why it was not running. You need to change the test numbers for all the query tests too.